packages/react-variant-view/src/util/AnnotationSummaryUtil.ts

```diff
@@ -0,0 +1,34 @@
+import {
+    TranscriptConsequenceSummary,
+    VariantAnnotationSummary,
+} from 'genome-nexus-ts-api-client';
+
+export function getTranscriptConsequenceSummary(
+    data: VariantAnnotationSummary | undefined
+): TranscriptConsequenceSummary {
+    let transcriptConsequenceSummary: TranscriptConsequenceSummary = {
+        aminoAcidAlt: '',
+        aminoAcidRef: '',
+        aminoAcids: '',
+        codonChange: '',
+        consequenceTerms: '',
+        entrezGeneId: '',
+        exon: '',
+        hgvsc: '',
+        hgvsp: '',
+        hgvspShort: '',
+        hugoGeneSymbol: '',
+        proteinPosition: {
+            start: 0,
+            end: 0,
+        },
+        refSeq: '',
+        transcriptId: '',
+        variantClassification: '',
+    };
+
+    if (data !== undefined && data.transcriptConsequenceSummary) {
+        transcriptConsequenceSummary = data.transcriptConsequenceSummary;
+    }
+    return transcriptConsequenceSummary;
```

Comment:
As a workaround for the TS error, we can do this for now:

```suggestion
    return transcriptConsequenceSummary as TranscriptConsequenceSummary;
```